src/pybind/ceph_volume_client.py

```diff
@@ -731,7 +731,7 @@ def purge_volume(self, volume_path, data_isolated=False):
             return
 
         def rmtree(root_path):
-            log.debug("rmtree {0}".format(root_path))
+            log.debug(u"rmtree {0}".format(root_path))
```

Comment:
i'd suggest drop this change. as in python3, `"rmtree {0}"` is a unicode string. and we've dropped python2 support.

Reply:
could you instead retarget this change to nautilus where we still support python2?

Reply:
Moved to https://github.com/ceph/ceph/pull/36679, shall we close this one?

Reply:
Yes: https://github.com/ceph/ceph/pull/36679#issuecomment-694609332